ee/vellum_ee/workflows/display/nodes/vellum/conditional_node.py

```diff
@@ -242,9 +246,9 @@ def _convert_descriptor_to_operator(self, descriptor: BaseDescriptor) -> str:
             return "doesNotBeginWith"
         elif isinstance(descriptor, DoesNotEndWithExpression):
             return "doesNotEndWith"
-        elif isinstance(descriptor, IsNullExpression):
+        elif isinstance(descriptor, (IsNullExpression, IsNilExpression, IsUndefinedExpression)):
             return "null"
-        elif isinstance(descriptor, IsNotNullExpression):
+        elif isinstance(descriptor, (IsNotNullExpression, IsNotNilExpression, IsNotUndefinedExpression)):
```

Comment:
We likely should do the same special casing here - but am deferring that for now to prioritize getting pull + run going for headspace